frontend/src/redux/stores/store.tsx

```diff
@@ -6,6 +6,12 @@ import reducers from '../reducers/reducers';
 import { INITIAL_STATE as UI_INITIAL_STATE } from '../reducers/ui';
 import rootSaga from '../sagas/sagas';
 
+window.config = {
```

Comment:
I was supposed to delete the one in the App.tsx but forgot. I had it there initially but moved it to the store because this way we get it in the tests as well.